src/api/aaregApi.ts

```diff
@@ -17,15 +17,16 @@ import {mapOrganisasjonerFraLowerCaseTilupper} from "./altinnApi";
 export async function hentArbeidsforholdFraAAregNyBackend(
     underenhet: string,
     enhet: string,
-    signal: any
+    signal: any,
+    erTidligereArbeidsforhold: boolean
 ): Promise<ObjektFraAAregisteret> {
     let headere = new Headers();
     headere.set('orgnr', underenhet);
     headere.set('jurenhet', enhet);
```

Comment:
de linjene her for å sette headere kan kankskje være en egen funksjon, ser de blir bruktpå linje 47-49 også